examples/ucp_hello_world.c

```diff
@@ -281,6 +294,12 @@ static int run_ucx_client(ucp_worker_h ucp_worker)
             CHKERR_JUMP(status != UCS_OK, "test_poll_wait\n", err_ep);
         }
     }
+    
+    if (err_handling_opt.failure_mode == FAILURE_MODE_KEEPALIVE) {
+        fprintf(stderr, "Emulating unexpected failure on client side: "
```

Comment:
Emulating unexpected failure after receive completion on client side, server should detect error by keepalive mechanism